### PR TITLE
VideoPress: use core lib to handle html entities

### PR DESCRIPTION
**Pull request by retrofox**
Follow-up of https://github.com/Automattic/jetpack/pull/28341

Instead of creating a new helper function to handle HTML entities, this PR:

* adds the HTML entities core package,
* updates the code that decodes the `title` and `description` of the block, 
* and finally removes the duplicated code. I've re-invented the wheel again.

Fixes #

## Proposed changes:

* VideoPress: use core lib to handle html entities

### Other information:

- [ ] Have you tested your changes on WordPress.com, if applicable (if so, you'll see a generated comment below with a script to run)?

## Testing instructions:

* Go to block editor
* Create/Edit a video block
* Set title and description
* Confirm data is saved
* Hard refresh
* Confirm data correct

### CODE REVIEW

**Comment by nunyvega** [comment]
Code changes look good, testing now

File: pnpm-lock.yaml [hunk 1]
```diff
@@ -4924,6 +4926,7 @@ packages:
   /@colors/colors/1.5.0:
     resolution: {integrity: sha512-ooWCrlZP11i8GImSjTHYHLkvFDP48nS4+204nGb1RiX/WXYHmJA2III9/e2DWVabCESdW7hBAEzHRqUn9OUVvQ==}
     engines: {node: '>=0.1.90'}
+    requiresBuild: true
```
Comment (nunyvega) [inline pnpm-lock.yaml:4929]: Why did this change?

Comment (retrofox) [reply pnpm-lock.yaml:4929]: I don't know. It's automatically generated by pnpm

File: projects/packages/videopress/changelog/update-videopress-use-core-pkg-to-handle-entities [hunk 2]
```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: changed
+
+VideoPress: use core lib to handle html entities
```
Comment (tyxla) [inline projects/packages/videopress/changelog/update-videopress-use-core-pkg-to-handle-entities]: I'd suggest being more specific about which package we're using here:

```suggestion
VideoPress: use @wordpress/html-entities to handle html entities
```

**Review by tyxla — DISMISSED** [review]
Looking great! Nice to hear that an existing `@wordpress` package solved it for your use case 👍

**Review by nunyvega — APPROVED** [review]
Tested and works as expected
 🚢

**Comment by retrofox** [comment]
> Looking great! Nice to hear that an existing @wordpress package solved it for your use case 👍

Yeah, me too! I forgot it last Friday 🤦

**Comment by tyxla** [comment]
> Yeah, me too! I forgot it last Friday 🤦

That makes us two 🙈

**Pull request merged**